src/main/resources/META-INF/rewrite/sonar.yml

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2020 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.testing.sonar.SonarBestPractices
+displayName: Sonar Best Practices for Java Testing Frameworks
+description: Sonar Best Practices for Java Testing Frameworks.
+tags:
+  - sonar
+  - testing
+recipeList:
+  - org.openrewrite.java.testing.sonar.TestsShouldIncludeAssertions:
+      assertionPackages:
+        - org.junit.jupiter.api.Assertions
+        - org.hamcrest.MatcherAssert
```

Comment:
We should definitely include AssertJ assertions in this list. testing-frameworks includes recipes for adopting AssertJ. It would be unfortunate to also ship a recipe unaware of them